Fixed handling of inline strings in XLSX

mweimerskirch opened the pull request:
This is:

```
- [x] a bugfix
- [ ] a new feature
```

Checklist:

- [ ] Changes are covered by unit tests
- [ ] Code style is respected
- [ ] Commit message explains **why** the change is made (see https://github.com/erlang/otp/wiki/Writing-good-commit-messages)
- [ ] CHANGELOG.md contains a short summary of the change
- [ ] Documentation is updated as necessary

### Why this change is needed?

I was working with inline strings and noticed that the cells showed up empty in both LibreOffice and Excel.
Upon inspecting the generated XML code, I noticed that it looked like this:
```xml
<c r="A1" t="inlineStr">
    <t>Test</t>
</c>
```

instead of this:
```xml
<c r="A1" t="inlineStr">
    <is>
        <t>Test</t>
    </is>
</c>
```

To fill the cells, I used $cell->setValueExplicit('Test', DataType::TYPE_INLINE);

After applying my patch, the cells appear correctly filled in LibreOffice and Excel.

Official documentation from Microsoft: https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.spreadsheet.inlinestring?view=openxml-2.8.1

This might solve #353. Not sure if it's exactly the same issue I was having.

oleibman commented:
It would be nice to include some evidence that your change works correctly. It is difficult to do this with a unit test, since the old code produces a syntactically correct spreadsheet; the missing data is evident only when the spreadsheet is opened. Instead, might you be able to add a new row using inlineStr to samples/Basic/02_Types.php? Note that this won't work initially for Xls; you need to make a one-line addition to Writer/Xls/Worksheet.php to add support for inline strings there.

mweimerskirch commented:
Here you go. I added an example value to the 02_Types.php file.
I also added "support" for inline strings in XLS, by handling them the exact same way as regular strings.

If you run the sample file before and after my patch, you can see that in the unpatched version, the C20 cell is empty while in the patched version it appears filled, like in the following screenshot:

![image](https://user-images.githubusercontent.com/362092/153627217-70c6a495-0d77-43a8-8878-3d6ad7545e19.png)

oleibman commented:
Thank you for your contribution.
